functions/http/index.js

```diff
@@ -148,20 +148,31 @@ exports.uploadFile = (req, res) => {
   if (req.method === 'POST') {
     const busboy = new Busboy({ headers: req.headers });
 
+    // This object will accumulate all the fields, keyed by their name
+    const fields = {};
+
     // This object will accumulate all the uploaded files, keyed by their name.
     const uploads = {};
     const tmpdir = os.tmpdir();
```

Comment:
I would add an empty line before:
const tmpdir = os.tmpdir();

Otherwise it looks like the comment "This object will accumulate all the uploaded files, keyed by their name" is for the two lines.

You can group this "const tmpdir" together with  "const busboy"